R/features.R

```diff
@@ -414,6 +414,337 @@ setMethod("calculatePeakQualities", "features", function(obj, weights, flatnessF
 })
 
 
+#' @describeIn features Obtain the total ion chromatogram/s (TICs) of the analyses.
+#' @param msLevels Integer vector with the ms levels (i.e., 1 for MS1 and 2 for MS2) 
+#' to obtain TIC traces.
+#' @aliases getTICs
+#' @export
+setMethod("getTICs", "features", function(obj, i = NULL, retentionRange = NULL, msLevels = 1)
+{
+  ac <- checkmate::makeAssertCollection()
+  
+  msLevels <- as.integer(msLevels)
+  checkmate::assert(checkmate::check_integer(msLevels, min.len = 1, lower = 1), add = ac)
+  
+  anaInfo <- analysisInfo(obj)
+  
+  if (!is.null(i))
+  {
+    if (is.character(i)) {
+      checkmate::assert(checkmate::check_choice(i, anaInfo$analysis), add = ac)
```

Comment:
No need to call `checkmate::assert()` --> `checkmate::assertChoice(i, anaInfo$analysis, add = ac)`

Reply:
ok, did not know.